lib/brook/storage/postgres/statement.ex

```diff
@@ -60,9 +60,10 @@ defmodule Brook.Storage.Postgres.Statement do
     ~s|
     CREATE TABLE IF NOT EXISTS #{table}
     (
-      key VARCHAR PRIMARY KEY,
       collection VARCHAR,
-      value JSONB
+      key VARCHAR,
+      value JSONB,
+      UNIQUE (collection, key)
```

Comment:
```suggestion
      PRIMARY KEY (collection, key)
```
This would be a combination of `unique` _and_ `not null`

Reply:
primary key also creates an btree index automatically so you may want to manually add an additional index or two in order to keep queries efficient

Reply:
Good point. I don't know why it didn't occur to me to use `PRIMARY KEY` there.